kotlin-spark-api/3.2/src/test/kotlin/org/jetbrains/kotlinx/spark/api/ApiTest.kt

```diff
@@ -593,10 +598,63 @@ class ApiTest : ShouldSpec({
                     it.nullable() shouldBe true
                 }
             }
+            should("Easily convert a (Java)RDD to a Dataset") {
```

Comment:
I think this huge test should be split into smaller ones?

Reply:
done